minio_operator_ext/operator.py

```diff
@@ -625,7 +615,11 @@ async def delete_minio_policy(self, resource: resources.MinioPolicy, **kwargs):
         async with self.create_minio_admin_client(policy.tenant) as minio_admin_client:
 
             def inner():
-                minio_admin_client.policy_remove(policy.name)
+                try:
+                    minio_admin_client.policy_remove(policy.name)
+                except minio.error.MinioAdminException as e:
+                    if e._code != "404":
```

Comment:
_Technically_, the minio admin client does _not_ return an error when the policy no longer exists - though this seems like an oversight.  For consistency with other resource deletion code paths, we should disregard 404s.